app/templates/server/config/environment/production.js

```diff
@@ -18,7 +18,11 @@ module.exports = {
     uri:    process.env.MONGOLAB_URI ||
             process.env.MONGOHQ_URL ||
             process.env.OPENSHIFT_MONGODB_DB_URL +
-            process.env.OPENSHIFT_APP_NAME ||
+            process.env.OPENSHIFT_APP_NAME ||<% if (filters.docker) { %>
+            (process.env.MONGODB_PORT_27017_TCP_ADDR ? 'mongodb://' +
+              process.env.MONGODB_PORT_27017_TCP_ADDR + ':' +
+              process.env.MONGODB_PORT_27017_TCP_PORT +
+              '/<%= _.slugify(appname) %>' : null) ||<% } %>
```

Comment:
This is code to make the docker node container see the mongo container https://www.thachmai.info/2015/05/10/docker-container-linking-mongo-node/